ompi/mca/osc/rdma/osc_rdma.h

```diff
@@ -251,8 +256,12 @@ struct ompi_osc_rdma_module_t {
     opal_mutex_t peer_lock;
 
 
-    /** BTL in use */
-    struct mca_btl_base_module_t *selected_btl;
+    /** BTL(s) in use */
+    struct mca_btl_base_module_t *selected_btls[MCA_OSC_RDMA_MAX_BTLS];
```

Comment:
Can you add a comment which btl is used for what? It looks like memory is registered with `selected_btls[0]`  but what is the second btl for?

Reply:
Sure. Will add a comment. Memory registration is only supported if one BTL is in use. This helps simplify the logic.

Reply:
Done.